[2023-05-01 10:16:41,042] INFO _log() L224   * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.35.33:5000 (Press CTRL+C to quit) - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-01 10:16:41,050] INFO _log() L224   * Restarting with stat - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-01 10:16:50,850] INFO _log() L224  127.0.0.1 - - [01/May/2023 10:16:50] "[32mGET / HTTP/1.1[0m" 302 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-01 10:16:50,872] INFO _log() L224  127.0.0.1 - - [01/May/2023 10:16:50] "GET /openapi/ HTTP/1.1" 200 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-01 10:16:51,225] INFO _log() L224  127.0.0.1 - - [01/May/2023 10:16:51] "[36mGET /openapi/static/images/swagger.svg HTTP/1.1[0m" 304 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-01 10:16:51,228] INFO _log() L224  127.0.0.1 - - [01/May/2023 10:16:51] "[36mGET /openapi/static/images/redoc.svg HTTP/1.1[0m" 304 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-01 10:16:51,230] INFO _log() L224  127.0.0.1 - - [01/May/2023 10:16:51] "[36mGET /openapi/static/images/rapidoc.svg HTTP/1.1[0m" 304 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-01 10:16:52,369] INFO _log() L224  127.0.0.1 - - [01/May/2023 10:16:52] "GET /openapi/swagger HTTP/1.1" 200 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-01 10:16:52,395] INFO _log() L224  127.0.0.1 - - [01/May/2023 10:16:52] "[36mGET /openapi/static/css/swagger-ui.css HTTP/1.1[0m" 304 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-01 10:16:52,401] INFO _log() L224  127.0.0.1 - - [01/May/2023 10:16:52] "[36mGET /openapi/static/js/swagger-ui-bundle.js HTTP/1.1[0m" 304 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-01 10:16:52,402] INFO _log() L224  127.0.0.1 - - [01/May/2023 10:16:52] "[36mGET /openapi/static/js/swagger-ui-standalone-preset.js HTTP/1.1[0m" 304 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-01 10:16:52,635] INFO _log() L224  127.0.0.1 - - [01/May/2023 10:16:52] "GET /openapi/openapi.json HTTP/1.1" 200 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-01 10:17:02,738] INFO _log() L224  127.0.0.1 - - [01/May/2023 10:17:02] "GET /carros HTTP/1.1" 200 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-01 10:17:35,654] INFO _log() L224  127.0.0.1 - - [01/May/2023 10:17:35] "GET /carros HTTP/1.1" 200 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-01 10:18:01,615] WARNING add_carro() L63   Erro ao adicionar carro 'Opala', N√£o foi poss√≠vel salvar novo item :/ - call_trace=C:\Projetos\IPT\desenvolvimento-full-stack\desenvolvimento-full-stack-basico\carros\api\app.py L63  
[2023-05-01 10:18:01,617] INFO _log() L224  127.0.0.1 - - [01/May/2023 10:18:01] "[31m[1mPOST /carro HTTP/1.1[0m" 400 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-01 10:18:21,196] INFO _log() L224  127.0.0.1 - - [01/May/2023 10:18:21] "GET /carros HTTP/1.1" 200 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-01 10:19:33,875] INFO _log() L224  127.0.0.1 - - [01/May/2023 10:19:33] "[36mGET /openapi/static/css/swagger-ui.css HTTP/1.1[0m" 304 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-01 10:19:34,078] INFO _log() L224  127.0.0.1 - - [01/May/2023 10:19:34] "[33mGET /openapi/static/js/swagger-ui-bundle.js.map HTTP/1.1[0m" 404 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-01 10:19:34,082] INFO _log() L224  127.0.0.1 - - [01/May/2023 10:19:34] "[33mGET /openapi/static/js/swagger-ui-standalone-preset.js.map HTTP/1.1[0m" 404 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-01 10:19:34,240] INFO _log() L224  127.0.0.1 - - [01/May/2023 10:19:34] "[33mGET /openapi/static/css/swagger-ui.css.map HTTP/1.1[0m" 404 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-01 10:19:59,094] WARNING add_carro() L63   Erro ao adicionar carro 'Uno', N√£o foi poss√≠vel salvar novo item :/ - call_trace=C:\Projetos\IPT\desenvolvimento-full-stack\desenvolvimento-full-stack-basico\carros\api\app.py L63  
[2023-05-01 10:19:59,096] INFO _log() L224  127.0.0.1 - - [01/May/2023 10:19:59] "[31m[1mPOST /carro HTTP/1.1[0m" 400 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-01 10:20:22,120] ERROR log_exception() L1449 Exception on /carro [GET] - call_trace=C:\Python311\Lib\site-packages\flask\app.py L1449
Traceback (most recent call last):
  File "C:\Python311\Lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\flask\app.py", line 1519, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\flask_cors\extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
                                                ^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\flask\app.py", line 1517, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\flask\app.py", line 1503, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\flask_openapi3\scaffold.py", line 109, in wrapper
    resp = _do_wrapper(
           ^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\flask_openapi3\do_wrapper.py", line 135, in _do_wrapper
    response = func(**request_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Projetos\IPT\desenvolvimento-full-stack\desenvolvimento-full-stack-basico\carros\api\app.py", line 107, in get_carro
    logger.warning(f"Erro ao buscar carro '{carro_modelo}', {error_msg}")
                                            ^^^^^^^^^^^^
NameError: name 'carro_modelo' is not defined
[2023-05-01 10:20:22,128] INFO _log() L224  127.0.0.1 - - [01/May/2023 10:20:22] "[35m[1mGET /carro HTTP/1.1[0m" 500 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-01 10:20:22,271] INFO _log() L224  127.0.0.1 - - [01/May/2023 10:20:22] "[33mGET /favicon.ico HTTP/1.1[0m" 404 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-01 10:21:16,383] INFO _log() L224   * Detected change in 'C:\\Projetos\\IPT\\desenvolvimento-full-stack\\desenvolvimento-full-stack-basico\\carros\\api\\app.py', reloading - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-01 10:21:16,522] INFO _log() L224   * Restarting with stat - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-01 10:21:18,123] INFO _log() L224   * Detected change in 'C:\\Projetos\\IPT\\desenvolvimento-full-stack\\desenvolvimento-full-stack-basico\\carros\\api\\app.py', reloading - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-01 10:21:18,298] INFO _log() L224   * Restarting with stat - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-01 10:21:43,623] INFO _log() L224  127.0.0.1 - - [01/May/2023 10:21:43] "GET /carros HTTP/1.1" 200 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-01 10:22:24,225] INFO _log() L224  127.0.0.1 - - [01/May/2023 10:22:24] "[33mGET /openapi/static/css/swagger-ui.css.map HTTP/1.1[0m" 404 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-01 10:22:24,230] INFO _log() L224  127.0.0.1 - - [01/May/2023 10:22:24] "[33mGET /openapi/static/js/swagger-ui-bundle.js.map HTTP/1.1[0m" 404 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-01 10:22:24,234] INFO _log() L224  127.0.0.1 - - [01/May/2023 10:22:24] "[33mGET /openapi/static/js/swagger-ui-standalone-preset.js.map HTTP/1.1[0m" 404 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-01 10:22:29,902] WARNING add_carro() L63   Erro ao adicionar carro 'UnoMile', N√£o foi poss√≠vel salvar novo item :/ - call_trace=C:\Projetos\IPT\desenvolvimento-full-stack\desenvolvimento-full-stack-basico\carros\api\app.py L63  
[2023-05-01 10:22:29,903] INFO _log() L224  127.0.0.1 - - [01/May/2023 10:22:29] "[31m[1mPOST /carro HTTP/1.1[0m" 400 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-01 10:24:34,481] WARNING add_carro() L57   Erro ao adicionar carro 'UnoMile', Carro de mesmo nome j√° salvo na base :/ - call_trace=C:\Projetos\IPT\desenvolvimento-full-stack\desenvolvimento-full-stack-basico\carros\api\app.py L57  
[2023-05-01 10:24:34,482] INFO _log() L224  127.0.0.1 - - [01/May/2023 10:24:34] "[31m[1mPOST /carro HTTP/1.1[0m" 409 - - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-01 10:24:47,622] WARNING add_carro() L63   Erro ao adicionar carro 'UnoMile2', N√£o foi poss√≠vel salvar novo item :/ - call_trace=C:\Projetos\IPT\desenvolvimento-full-stack\desenvolvimento-full-stack-basico\carros\api\app.py L63  
[2023-05-01 10:31:54,116] INFO _log() L224   * Restarting with stat - call_trace=C:\Python311\Lib\site-packages\werkzeug\_internal.py L224 
